tools/lint-docstring

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+
+import argparse
+import glob
+import importlib
+from pathlib import Path
+from typing import Dict, List
+
+
+TABLE_OFFSET = 7
+COLUMN_WIDTHS = (23, 20, 103)
+SPACEBAR = " "
+
+# leaving blank rows after exhausting files from each folder
+BLANK_ROW = f"| {SPACEBAR:{COLUMN_WIDTHS[0]}}| {SPACEBAR:{COLUMN_WIDTHS[1]}}| {SPACEBAR:{COLUMN_WIDTHS[2]}}|\n"
+
+# absolute path to zulip-terminal
+ROOT_DIRECTORY = Path(__file__).resolve().parent.parent
+
+# absolute path to zulip-terminal/zulipterminal to be passed as parameter
+ZULIPTERMINAL = ROOT_DIRECTORY / "zulipterminal"
+
+# new doc file has been created for the time being to compare it with the original
+DEVELOPER_DOC = ROOT_DIRECTORY / "docs" / "developer-file-overview.md"
+
+# zulipterminal/scripts and zulipterminal/themes do not have any files
+# mentioned in the file-overview. Hence, their description has been
+# added separately.
+DOCSTRINGS_WITHOUT_FILES = {
```

Comment:
Was this name change to ensure everything fits on 1 line?
Maybe `DESC_FOR_NO_FILE_FOLDERS` is a better name?